server/game/cards/characters/05/serlancellannister.js

```diff
@@ -0,0 +1,29 @@
+const DrawCard = require('../../../drawcard.js');
+
+class SerLancelLannister extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => !this.getSingleOtherLannisterLordOrLady(),
```

Comment:
This condition seems wrong. Shouldn't it evaluate truthy when there *is* another single Lannister lord or lady? This will evaluate true when there isn't one.